dataactbroker/handlers/account_handler.py

```diff
@@ -165,6 +170,79 @@ def max_login(self, session):
             # Return 500
             return JsonResponse.error(e, StatusCode.INTERNAL_ERROR, error=str(e))
 
+    def caia_login(self, session):
+        """ Logs a user in if their password matches using CAIA
```

Comment:
Minor thing but I don't think "password" is the right way to explain this. Perhaps something like "if their CAIA validation succeeds"